example/index.html

```diff
@@ -39,6 +55,45 @@ <h2>Scope</h2>
       <li><cite id="bib-ietf-rfc-8259">IETF RFC 8259</cite>, The JavaScript
       Object Notation (JSON) Data Interchange Format, T. Bray, <a>http://www.ietf.org/rfc/rfc8259.txt</a></li>
     </ul>
+
+  </section>
+  <!-- End of Normative references -->
+
+  <!-- Start of Terms and definitions - Do not remove this section -->
+  <section id="sec-terms-and-definitions">
+
+    <!-- Use the `ul` element to list external documents where Terms are defined (use same logic as referenced docs in the body of the document as example shows). Add `dt/dd` sub element pairs as needed for Terms to be defined internally in this document. Remove either the `ul` if no external Terms, or `dl` if no internally defined Terms. If no Terms needed to be defined at all, remove the both `ul` and `dl` elements. Pre text for this section is auto generated based on what elements are present. -->
+
+    <ul id="terms-ext-defs">
```

Comment:
There is no semantic indication that the members of the list contain terms and definitions that apply. Can we think of a way to conveying those semantics?

Reply:
Resolved in https://github.com/SMPTE/html-pub/pull/27/commits/164cbd6c669f72883acd258fbb2d8809377d5686